provisioning/roles/postgis/vars/Debian-14.yml

```diff
@@ -0,0 +1 @@
+postgis_package_name: "postgresql-9.3-postgis-2.1"
```

Comment:
Yes! I used the variable {{ ansible_os_family }}, where Ubuntu returns Debian, which is kind of cute because it means they are part of the same family... instead of using {{ ansible_distribution }}. That's fixed.